### ISSUE TITLE
sorry to bother you!

### ISSUE THREAD

**Issue by markofan** [Ectsang/3D-CNN-Keras#1]
SORRY! I have a problem, I am so sorry to bother you!
I use Keras to build my cnn, I set my Validation data in my model.fit function--use validation_spilt=0.25
But something strange happen!
My training accuracy is about 80%, but my validation accuracy is 100% at the begining, and it doesnt change, so I want to ask you, what is wrong with my cnn?
Thank you so much!
